elasticsearch_metrics/metric.py

```diff
@@ -64,11 +65,29 @@ def get_index_template(cls):
     @classmethod
     def get_index_name(cls, date=None):
         date = date or timezone.now().date()
-        # TODO: Make dateformat configurable
-        dateformat = "%Y.%m.%d"
+        dateformat = settings.DATE_FORMAT
         date_formatted = date.strftime(dateformat)
         return "{}-{}".format(cls._template_name, date_formatted)
 
 
 class Metric(Document, BaseMetric):
     __doc__ = BaseMetric.__doc__
+
+    @classmethod
+    def init(cls, index=None, using=None):
+        """
+        Create the index and populate the mappings in elasticsearch.
+        """
+        i = cls._index
+        name = index or cls.get_index_name()
+        i = i.clone(name=name)
+        i.save(using=using)
```

Comment:
Any reason not to do 

```python
return super(...).init(index=index or cls.get_index_name())
```

?

Reply:
ah, sorry, I had copied that over when I was trying to figure out if get_index_name should be overriding "i" or "index".